application-templates/javascript/assets/src/main.js

```diff
@@ -0,0 +1,21 @@
+import commercetoolsLogo from '../static/commercetools.svg';
```

Comment:
Let us import files only from the `src` folder, and then we specify the static folder as a public directory in Vite.

```typescript
export default defineConfig({
  // ...
  build: {
    publicDir: './static',
    outDir: './public',
  },
})

```